pages/apply/[resident]/summary.tsx

```diff
@@ -62,6 +97,7 @@ const UserSummary = (): JSX.Element => {
       <MedicalNeedsSummary currentResident={currentResident} />
 
       <ButtonLink href="/apply/overview">I confirm this is correct</ButtonLink>
+      <Button onClick={onConfirmData}>I confirm this is correct Button</Button>
```

Comment:
To be updated? duplicate buttons

Reply:
My bad. This has been removed

Reply:
sorry, was referring to what looks like `ButtonLink` that needs removing and `DeleteLink` reinstated to match designs.